db/src/logger.rs

```diff
@@ -320,11 +324,12 @@ impl LogMetadata {
     }
 }
 
+#[cfg(not(target_family = "wasm"))]
```

Comment:
is the idea of impl'ing drop incompatible with wasm or just the `unlock` statement? if it's the later then just be more surgical with the cfg